analysis_options.yaml

```diff
@@ -1,3 +1,4 @@
+include: package:pedantic/analysis_options.yaml
```

Comment:
We should peg this to a specific version so that changes to the pedantic package won't break the CI here.